js/game/pet.js

```diff
@@ -55,6 +55,10 @@ export class Pet extends GameObject {
     this.talk = false
     this.talkText = ''
 
+    this.hungerPoints = 10;
+    this.lastHungerDepletionTime = Date.now(); // ms; sets the last time the pet's hunger dropped 1 point
```

Comment:
I think we need to store this in the database. Right now this value will reset every time they re-open ComfyUI.  Will only work if they leave their ComfyUI on